src/main/java/build/buildfarm/instance/server/NodeInstance.java

```diff
@@ -1906,7 +1898,7 @@ public boolean pollOperation(String operationName, ExecutionStage.Value stage) {
 
   protected CacheCapabilities getCacheCapabilities() {
     return CacheCapabilities.newBuilder()
-        .addDigestFunctions(digestUtil.getDigestFunction())
+        .addAllDigestFunctions(DigestUtil.getSupportedDigestFunctions())
```

Comment:
Current change mostly focus on supporting multiple hashing algorithm for write requests but this change will signal client that buildfarm support multiple hashing algorithm for all apis including `ExecutionService.Execute` API.

Reply:
Then how would we make the change just for write? If get Capabilities doesn't say it accepts SHA and BLAKE, the client will cancel the request if it uses a different algorithm

Reply:
This could be internal patch until support for multiple hash function is enabled for all APIs.